src/python/elrs_helpers.py

```diff
@@ -46,18 +13,17 @@ def get_git_version(env):
     ver = "ver.unknown"
     sha = "000000"
 
-    git_repo = get_git(env)
-    if git_repo:
-        import git
-        sha = git_repo.head.object.hexsha
+    try:
+        call_git = lambda cmd: subprocess.check_output(["git"] + cmd).decode("utf-8").rstrip('\r\n')
+        sha = call_git(["symbolic-ref", "-q", "--short", "HEAD"])
```

Comment:
This does not return the SHA, should be `ver = call_git(["rev-parse", "HEAD"])`

Reply:
Indeed, this is a copy paste error - good catch!